### DISCUSSION TITLE
read macp file using foxglove call error:  Failed to initialize message preloading Time range is too long to be supported

### DISCUSSION THREAD

**Discussion by noone3001** [foxglove/community#1062] [Q&A]
**read macp file using foxglove call error:  Failed to initialize message preloading Time range is too long to be supported**

- Version:
- Platform:

**Steps To Reproduce**

record messagge bag using ros2 bag record -a;
paly  bag using foxglove;
error comes out :
*The start (1970-01-01T00:00:00.000000000Z) and end (2024-12-17T09:35:31.642321592Z) of your data is too far apart.
Failed to initialize message preloading
Time range is too long to be supporte*

**Expected Behavior**
data can be showed on foxglove

**Comment by linear** [comment]
<p><a href="https://linear.app/foxglove/issue/FG-9872/read-macp-file-using-foxglove-call-error-failed-to-initialize-message">FG-9872 read macp file using foxglove call error: Failed to initialize message preloading Time range is too long to be supported</a></p>

**Comment by noone3001** [comment]
> [FG-9872 read macp file using foxglove call error: Failed to initialize message preloading Time range is too long to be supported](https://linear.app/foxglove/issue/FG-9872/read-macp-file-using-foxglove-call-error-failed-to-initialize-message)
sorry I can not log in the cite

**Comment by jtbandes** [comment]
> The start (1970-01-01T00:00:00.000000000Z) and end (2024-12-17T09:35:31.642321592Z) of your data is too far apart.


Why does your data include timestamps from both 1970 and 2024?

**Reply by noone3001** [reply]
I used my PC record bag from a hardware in a test car. The env on my PC : wins11 with WSL, I used teminal window call WSL(Ubuntu subsystem in wins), then used ros2 command " ros2 bag record -a "  to record message bag. I also confused by the timestamp of 1970
